### PR TITLE
AST-966 - Test button presets with third party plugins like WooCommerce

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Description

Global color compatibility with WooCommerce, LearnDash, LifterLMS
- Border width
- Button Text + BG colors
- Border radius 

### Screenshots
Woocommerce:- https://share.bsf.io/o0u7YAzG

### Types of changes
New improvements.

### How has this been tested?
Added test cases in the card.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by premanshup** [comment]
@GauravKhupse Do sync this PR.

File: changelog.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ v3.8.0 (woocomerce-improvements) (Unreleased)
 - New: WooCommerce - Added cart Margin and Padding spacing option.
 - Improvement: Added a new filter 'astra_web_stories_position' to change the position of web stories.
 - Fix: Responsive color options from 'color-group-control' are hiding when trying to set them in responsive devices.
+- Fix: Fixed Button settings options are not working in customizer preview for WooCommerce buttons.
```
Comment (imnavanath) [inline changelog.txt:9]:
```suggestion
- Fix: Global button design options are not working in customizer preview for WooCommerce & LifterLMS buttons.
```

Comment (GauravKhupse) [reply changelog.txt]: Changes done!

File: changelog.txt [hunk 2]
```diff
@@ -6,6 +6,7 @@ v3.8.0 (woocomerce-improvements) (Unreleased)
 - New: WooCommerce - Added cart Margin and Padding spacing option.
 - Improvement: Added a new filter 'astra_web_stories_position' to change the position of web stories.
 - Fix: Responsive color options from 'color-group-control' are hiding when trying to set them in responsive devices.
+- Fix: Global button design options are not working in customizer preview for WooCommerce & LifterLMS buttons.
```
Comment (premanshup) [inline changelog.txt:9]:
```suggestion
- Fix: Global button design options are not working in customizer preview for WooCommerce buttons.
```

**Review by premanshup — APPROVED** [review]

File: changelog.txt (hunk 2 above)
Comment (GauravKhupse) [reply changelog.txt]: Changes done!

**Review by imnavanath — APPROVED** [review]

**Pull request merged**